main.go

```diff
@@ -0,0 +1,47 @@
+/*
+ *Author: Stefan
+ *Date: 11/16/2019
+ *Last changes: 11/18/2019 15.09
+ *Task: Implement the following methods: End - defines the end of the shape, Perimiter, Area
+**/
+
+package main
+
+import "fmt"
+
+//Point structure
+type Point struct {
+	x, y int
+}
+
+//Square structure
+type Square struct {
+	start Point
+	a     uint
+}
+
+//End method
+func (p *Square) End() (q Point) { //  func (p *Square) End() Point {
+	var xend = p.start.x + int(p.a) //	var xend = p.start.x + int(p.a)
+	var yend = p.start.y - int(p.a) //	var yend = p.start.y - int(p.a)
```

Comment:
```suggestion
	q.y = p.start.y - int(p.a) //	var yend = p.start.y - int(p.a)
```

Reply:
I edited the style. 
But in fact, these changes only affect readability and are not an error. Correctly?

Reply:
Correct.